### PR TITLE
Update package.json for fork (scoped/prefixed package name, updated project URLs)

### PR DESCRIPTION
**Pull request by DeeDeeG**
### Description of the Change

Change the package name to be the scoped/prefixed `@atom-ide-community/atom-package-manager`.

(If this is the wrong prefix, please let me know.)

### Alternate Designs

We could use some other scope/prefix, rather than `@atom-ide-community`.

### Benefits

Allows us to publish to the npmjs.com package registry, clearly distinguishes our source tree from upstream's.

### Possible Drawbacks

None, unless I'm misunderstanding and we can publish to the scoped name without changing `package.json`... In which case this might be unnecessary, but I'd still like to do this to distinguish our source tree from upstream's.

### Verification Process

Compare with how `@rails/webpacker` does this in their `package.json`:
https://github.com/rails/webpacker/blob/4fbb7bcadcd190b2c21cfc3c6bd894328e648e65/package.json#L2

### Applicable Issues

N/A

### CODE REVIEW

**Comment by DeeDeeG** [comment]
@aminya I made some PRs at this fork.

Please take a look if you have the time, or if you prefer I can handle merging. I can basically take care of/be the main maintainer of this repo myself if you like.

---

If you end up merging any of these, please merge this PR _FIRST_. I want our history of changes to always have a `package.json` that reflects the fact that we've forked from upstream.

Friendly reminder: If we are going to use a different prefix than `@atom-ide-community`, then less discuss that before merging this, of course. I'm okay with that prefix, personally, but if there's consensus that's even better.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "atom-package-manager",
+  "name": "@atom-ide-community/atom-package-manager",
```
Comment (aminya) [inline package.json:2]: Please update the `url` in `package.json` as well.

Comment (DeeDeeG) [reply package.json:2]: Good catch.

Comment (DeeDeeG) [reply package.json:2]: We don't have issues enabled here, so the "bugs" URL would point to nowhere in particular if we filled it out. Maybe I should point it to our "new pull request" URL...

Consider enabling issues here (I personally can't do that, apparently).

Comment (DeeDeeG) [reply package.json:2]: When you link to https://github.com/atom-ide-community/apm/issues, and the issues aren't enabled for the repo, it _automatically_ redirects users to the ["pull requests" page](https://github.com/atom-ide-community/apm/pulls).

Kinda weird user experience, but it's much better than a 404, and it's what I was considering doing anyway. Other option is to delete the "bugs" object from the package.json entirely.

File: package.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   "description": "Atom package manager",
```
Comment (DeeDeeG) [inline package.json:3]: "Community build of the Atom package manager"?

Comment (aminya) [reply package.json:3]: Yes, that's a good idea.

Comment (aminya) [inline package.json:3]:
```suggestion
  "description": "Community build of the Atom package manager",
```

**Comment by DeeDeeG** [comment]
Hi, thanks for looking at/reviewing/updating this.

If you can, I'd like to not merge master into our PRs. It makes the commit history more complicated, and I try to make neat PRs, so while it doesn't mean a lot, it kinda makes me sad to see the extra noise.

ANYHOW, we are making good progress.

**Comment by DeeDeeG** [comment]
Oh, and thanks for activating CI!! :heart:

**Pull request merged**